spec/acceptance/test_unit_spec.rb

```diff
@@ -211,6 +211,77 @@
       end
     end
 
+    context 'with unbalanced json in the test descriptions' do
+      before(:all) do
+        FileUtils.mkdir_p(File.join('spec', 'unit'))
+        File.open(File.join('spec', 'unit', 'unbalanced_json_spec.rb'), 'w') do |f|
+          f.puts <<-EOF
+            RSpec.describe "broken-junit" do
+              let(:large_nested_hash) do
+                {
+                  single_one: 'single_one',
+                  single_two: 'single_two',
+                  single_three: 'single_three',
+                  single_four: 'single_four',
+                  single_five: 'single_five',
+                  nested: {
+                    nested_one: 'nested_one',
+                    nested_two: 'nested_two',
+                    nested_three: 'nested_three',
+                    nested_four: 'nested_four',
+                    nested_five: 'nested_five',
+                  },
+                }
+              end
+
+              subject do
+                large_nested_hash
+              end
+
+              it { is_expected.to be_a Hash }
+              it { is_expected.to eq large_nested_hash }
+            end
+          EOF
+        end
+      end
+
+      after(:all) do
+        FileUtils.rm_rf(File.join('spec', 'unit'))
```

Comment:
This seems dangerous. What's cwd at this point?

Reply:
cwd is the test module directory, but i can replace this with a couple of safer methods

Reply:
👍 I imagine my face if this ever went hay-wire would be 😕💥😢

Absolute path would be nice but that could be hard to get 🤷‍♂️